e2e/nomostest/clusters/kind.go

```diff
@@ -136,6 +136,11 @@ func (c *KindCluster) Connect() error {
 	return c.provider.ExportKubeConfig(c.Name, c.KubeConfigPath, false)
 }
 
+// Hash returns empty string for kind cluster
+func (c *KindCluster) Hash() (string, error) {
+	return "", nil
```

Comment:
maybe something like "<KinD>" or something so it's not just empty when logged.

Reply:
done